content/en/docs/tasks/traffic-management/mirroring/index.md

```diff
@@ -235,7 +235,7 @@ log entries for `v1` and none for `v2`:
     EOF
     {{< /text >}}
 
-    This route rule sends 100% of the traffic to `v1`. The last stanza specifies
+    This route rule sends 100% of the traffic from `v1` to `v2`. The last stanza specifies
```

Comment:
This change is not correct. As documented, the route rule is sending 100% of the traffic to the `v1` version of httpbin. See the fields `weight: 100` and `subset: v1`. In addition it's also mirroring 100% of the traffic to `v2`. That's not the same as sending the v1 traffic to v2.